### ISSUE TITLE
Google Analytics update from 1.0.5 to 1.0.6

### ISSUE THREAD

**Issue by marcoelho97** [rainlab/googleanalytics-plugin#34]
Hi, as I got in this morning, I saw a new update to the plugin, so, I decided to update! unfortunatly, I got an error that crashed my whole website! Frontend and backend! I had to replace the google analytics new files for the old ones in order to get it to work.

I did try in a local machine again, to see if it would work, and it did, so, I don't know what's wrong... Here's the error:

```
<!DOCTYPE html> <html> <head> <meta charset="UTF-8" /> <meta name="robots" content="noindex,nofollow" /> <style> /* Copyright (c) 2010, Yahoo! Inc. All rights reserved. Code licensed under the BSD License: http://developer.yahoo.com/yui/license.html */ html{color:#000;background:#FFF;}body,div,dl,dt,dd,ul,ol,li,h1,h2,h3,h4,h5,h6,pre,code,form,fieldset,legend,input,textarea,p,blockquote,th,td{margin:0;padding:0;}table{border-collapse:collapse;border-spacing:0;}fieldset,img{border:0;}address,caption,cite,code,dfn,em,strong,th,var{font-style:normal;font-weight:normal;}li{list-style:none;}caption,th{text-align:left;}h1,h2,h3,h4,h5,h6{font-size:100%;font-weight:normal;}q:before,q:after{content:'';}abbr,acronym{border:0;font-variant:normal;}sup{vertical-align:text-top;}sub{vertical-align:text-bottom;}input,textarea,select{font-family:inherit;font-size:inherit;font-weight:inherit;}input,textarea,select{*font-size:100%;}legend{color:#000;} html { background: #eee; padding: 10px } img { border: 0; } #sf-resetcontent { width:970px; margin:0 auto; } .sf-reset { font: 11px Verdana, Arial, sans-serif; color: #333 } .sf-reset .clear { clear:both; height:0; font-size:0; line-height:0; } .sf-reset .clear_fix:after { display:block; height:0; clear:both; visibility:hidden; } .sf-reset .clear_fix { display:inline-block; } .sf-reset * html .clear_fix { height:1%; } .sf-reset .clear_fix { display:block; } .sf-reset, .sf-reset .block { margin: auto } .sf-reset abbr { border-bottom: 1px dotted #000; cursor: help; } .sf-reset p { font-size:14px; line-height:20px; color:#868686; padding-bottom:20px } .sf-reset strong { font-weight:bold; } .sf-reset a { color:#6c6159; cursor: default; } .sf-reset a img { border:none; } .sf-reset a:hover { text-decoration:underline; } .sf-reset em { font-style:italic; } .sf-reset h1, .sf-reset h2 { font: 20px Georgia, "Times New Roman", Times, serif } .sf-reset .exception_counter { background-color: #fff; color: #333; padding: 6px; float: left; margin-right: 10px; float: left; display: block; } .sf-reset .exception_title { margin-left: 3em; margin-bottom: 0.7em; display: block; } .sf-reset .exception_message { margin-left: 3em; display: block; } .sf-reset .traces li { font-size:12px; padding: 2px 4px; list-style-type:decimal; margin-left:20px; } .sf-reset .block { background-color:#FFFFFF; padding:10px 28px; margin-bottom:20px; -webkit-border-bottom-right-radius: 16px; -webkit-border-bottom-left-radius: 16px; -moz-border-radius-bottomright: 16px; -moz-border-radius-bottomleft: 16px; border-bottom-right-radius: 16px; border-bottom-left-radius: 16px; border-bottom:1px solid #ccc; border-right:1px solid #ccc; border-left:1px solid #ccc; } .sf-reset .block_exception { background-color:#ddd; color: #333; padding:20px; -webkit-border-top-left-radius: 16px; -webkit-border-top-right-radius: 16px; -moz-border-radius-topleft: 16px; -moz-border-radius-topright: 16px; border-top-left-radius: 16px; border-top-right-radius: 16px; border-top:1px solid #ccc; border-right:1px solid #ccc; border-left:1px solid #ccc; overflow: hidden; word-wrap: break-word; } .sf-reset a { background:none; color:#868686; text-decoration:none; } .sf-reset a:hover { background:none; color:#313131; text-decoration:underline; } .sf-reset ol { padding: 10px 0; } .sf-reset h1 { background-color:#FFFFFF; padding: 15px 28px; margin-bottom: 20px; -webkit-border-radius: 10px; -moz-border-radius: 10px; border-radius: 10px; border: 1px solid #ccc; } </style> </head> <body> <div id="sf-resetcontent" class="sf-reset"> <h1>Whoops, looks like something went wrong.</h1> <h2 class="block_exception clear_fix"> <span class="exception_counter">1/1</span> <span class="exception_title"><abbr title="Symfony\Component\Debug\Exception\FatalErrorException">FatalErrorException</abbr> in <a title="/home/mywebsite/public_html/plugins/october/drivers/vendor/guzzlehttp/guzzle/src/functions.php line 265" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">functions.php line 265</a>:</span> <span class="exception_message">Cannot redeclare GuzzleHttp\uri_template() (previously declared in /home/mywebsite/public_html/plugins/rainlab/googleanalytics/vendor/guzzlehttp/guzzle/src/functions.php:18)</span> </h2> <div class="block"> <ol class="traces list_exception"> <li> in <a title="/home/mywebsite/public_html/plugins/october/drivers/vendor/guzzlehttp/guzzle/src/functions.php line 265" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">functions.php line 265</a></li> </ol> </div> </div> </body> </html>
```

And that is litteraly the error that shows up, html and all, I did not copy the code, only the error that I was showed.

After the error, both frontend and backend show this error:
![screenshot_4](https://cloud.githubusercontent.com/assets/13586621/12230757/66740d38-b84a-11e5-819b-57be15286c6d.png)

**Comment by daftspunk** [comment]
The GA library appears to keep changing, what version of PHP are you using?

**Comment by marcoelho97** [comment]
I have tried with both 5.4 and 5.6, both failed.

**Comment by daftspunk** [comment]
I can't seem to replicate the issue on my end. What other plugins do you have installed?

**Comment by marcoelho97** [comment]
There are it:
Location
Sitemap
Pages
Twitter
Google Analytics
MailChimp
Blog
User
SEO Extension
Photobox
Backend secure by IP
Translate
Drivers
Flexi Contact
Company
Youtube Videos
MyCalendar

**Comment by marcoelho97** [comment]
Is there any php module that needs to be active??

**Comment by daftspunk** [comment]
Looks like this is an issue with composer, even though we reference a commit with no dependencies, it still downloads the dependencies from the latest commit. Very odd.

So it means Guzzle conflicts with the October.Drivers plugins . The issue has been fixed in 408bdd07baf55a3e73c612ebebce9fc655c48d51

Please uninstall and reinstall the GA plugin.

**Comment by marcoelho97** [comment]
That fixed it! Thanks!! Cheers
